includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -116,7 +118,7 @@ public function setup_wizard() {
 		) );
 
 		if ( ! empty( $_POST['save_step'] ) && isset( $this->steps[ $this->step ]['handler'] ) ) {
-			call_user_func( $this->steps[ $this->step ]['handler'] );
+			call_user_func( $this->steps[ $this->step ]['handler'], $this->get_next_step_link() );
```

Comment:
That's a better idea, IMO. I'll just pass `$this` in instead, and change the method to `get_next_step_link( $step = '' )` so it can get the current next step by default or any other next step.

@mikejolley @claudiosanches any problems you notice with that approach? I think that should work well.